src/main.cpp

```diff
@@ -29,6 +29,38 @@ void initialize_pins()
 	}
 }
 
+float calculate_frequency(double velocity)
+{
+	// Take the equation the propulsion subteam provided, substitute the
+	// constants, and simplify. The resulting equation is a quadratic, where
+	// x is the frequency and the velocity is a constant. This means that
+	// we can use the quadratic formula and solve for the appropriate frequency.
+	// Note that since this is a quadratic, there are two solutions. We
+	// default to the solution that has the higher frequency for increased
```

Comment:
Chore: comment needs to be updated
```suggestion
	// default to the solution that has the lower frequency for increased
```